ci/install-pktgen/pktgen-config.lua

```diff
@@ -0,0 +1,111 @@
+--                    openNetVM
```

Comment:
Rename this to something like pktgen-timed-cfg.lua

Reply:
I can but I'll just write in a comment that it's going to rename it in the script, because otherwise we would need to make a new `run-pktgen.sh` script to run with the new named lua config